app/javascript/components/exercise/TheProblems.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import ProblemContents from './ProblemContents'
+
+class TheProblems extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      visibleProblemIndex: 0
+    }
+  }
+  renderTabs = (visibleProblemIndex) => {
+    return (
+      <div className='tabs'>
+        {
```

Comment:
**[eslint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Replace `⏎··········this.props.problemList.map((p,·i)·=>·(⏎············this.renderTab(i,·visibleProblemIndex)⏎··········))⏎········` with `this.props.problemList.map((p,·i)·=>·this.renderTab(i,·visibleProblemIndex))` (prettier/prettier)